hack/ci/link-check.sh

```diff
@@ -2,6 +2,6 @@
 set -ev
 
 docker volume create olm-html
-docker run --rm -v "$(pwd):/src" -v olm-html:/src/public klakegg/hugo:0.73.0-ext-ubuntu
+docker run --rm --privileged=true -v "$(pwd):/src" -v olm-html:/src/public:Z klakegg/hugo:0.73.0-ext-ubuntu
```

Comment:
@anik120 It looks like we need to update the README instructions for running this check locally for boxes that have these filesystems owned by root.

You likely wouldn't want to run this as a privileged container but documenting ways to overcome any potential permission errors when mounting that volume would be a step in the right direction.

Reply:
Pulled out those changes into their own PR. I'll continue messing around with fixing those external/internal link errors that the linting script is producing.

Reply:
Ended up pulling out the changes needed to get CI green again in #101.